src/services/users.services.js

```diff
@@ -0,0 +1,43 @@
+const { User } = require('../models/User.model');
+
+const getUsers = async () => {
+  const users = await User.findAll({ order: ['name'] });
+
+  return users;
+};
+
+const getUserById = async (id) => {
+  const user = await User.findByPk(id);
+
+  return user;
+};
+
+const createUser = async (name) => {
+  const user = await User.create({ name: name });
```

Comment:
Consider adding error handling for the `create` operation to manage potential database errors, such as validation errors or connection issues.